recipe/meta.yaml

```diff
@@ -41,19 +41,22 @@ requirements:
     - conda-package-handling >=1.3.0
     - menuinst >=1.4.11,<2         # [win]
     - pip
-    - ruamel_yaml >=0.11.14,<0.17
+    - ruamel.yaml >=0.11.14,<0.18
```

Comment:
@beckermr That's something you think should be way more constraint, right?

Reply:
It depends on your tolerance for risk, but for sure there will be a tradeoff between broken envs, envs that don't solve, and maintenance to update pins.

Reply:
I would at least make the minimum version as big as possible, and I'm not a huge fan of pessimistic upper version pins.

Reply:
I'd appreciate a specific proposal of change here.

Reply:
https://github.com/conda/conda/blob/main/conda/common/serialize.py#L47

Reply:
I like the changes here already.

Reply:
conda's serialize module appears to use the current YAML().method form always, instead of the deprecated global yaml.load_safe() or whatever.